code/procs/mob_procs.dm

```diff
@@ -1056,21 +1056,28 @@
 					G.visible_message("<span class='alert'><b>[src]</b> violently slashes [G]!</span>")
 				playsound(G.loc, "sound/impact_sounds/Metal_Hit_Light_1.ogg", 80, 1)
 				G.damage_slashing(15)
-				return 1
+				return TRUE
 
 		if (S == "door" && istype(target, /obj/machinery/door))
 			var/obj/machinery/door/door = target
 			SPAWN(0)
 				door.tear_apart(src)
-			return 1
+			return TRUE
 
 		if (S == "table" && istype(target, /obj/table))
 			var/obj/table/table = target
 			playsound(table.loc, "sound/impact_sounds/Generic_Hit_Heavy_1.ogg", 40, 1)
 			table.deconstruct()
-			return 1
-
-	return 0
+			return TRUE
+
+		if (S == "blob" && istype(target, /obj/blob))
+			var/obj/blob/B = target
+			if(show_message)
+				src.visible_message("<span class='alert'><B>[src] savagely slashes [B]!</span>", "<span class='notice'>You savagely slash at \the [B]</span>")
+			B.take_damage(rand(6,10),1,DAMAGE_CUT)
```

Comment:
tbh I'd probably bump this up to at least an average of 10 damage, maybe even a min of 10. It's fairly niche, and should probably be at least as good as using a fire extingisher 🤷